bin/regen_screenshots

```diff
@@ -146,8 +148,10 @@ class RegenScreenshots
     resize(1800)
     click_link "Compute"
     click_link "New Virtual Machine"
-    screenshot "quickstart/managed-services-2-screenshot.png"
+    screenshot "quick-start/managed-services-2-screenshot.png"
 
+# rubocop:disable Style/BlockComments
```

Comment:
I endorse eliminating this cop. It's not a convenient rewrite one gets for free, even if it was, one may not want it, and block comments are not the kind of thing people do accidentally without a reason.

Reply:
I added a commit to eliminate the cop by default.